### PR TITLE
docs update

### PR DESCRIPTION
**Pull request by dwaas**
INDEX
   - added troubleshooting section
   - TODO: upload mnt script
   - typos fix and wording changes

### CODE REVIEW

File: doc/index.md [hunk 1]
```diff
@@ -9,17 +9,20 @@ To build a new image, you have to:
...
 * be root
 * put the installation-images directory on a *local* file system (ie. not NFS)
+* have a valid .oscrc
 
-Then you can run ```make``` to build all parts.
+Then you must run ```make``` once to build all parts.
 
-The images are stored in the 'image' directory, the contents of the images
-is in the 'tmp' directory.
+The images will stored in the 'image' directory, the contents of the images
```
Comment (markkp) [inline doc/index.md]: Should this be "The images will be stored" instead of "The images will stored" ?

File: doc/configoptions.md [hunk 2]
```diff
@@ -110,7 +110,7 @@
 
 - linuxrc=&lt;absolute_filename&gt;
 
-  use the specified linuxrc (taken from the running system!)
+  use the specified linuxrc (taken from the running system!). Caveats may apply: please also refer to the [linuxrc docs](https://github.com/openSUSE/linuxrc) and the [troubleshooting section](#troubleshooting-/-hacks).
```
Comment (dwaas) [inline doc/configoptions.md]: the troubleshooting link is broken, need to investigate.

Comment (dwaas) [inline doc/configoptions.md]: fixed in the latest refactoring

**Comment by dwaas** [comment]
General documentation update. 


NOTE: the mnt script will be eventually referenced in other pull requests.

**Pull request merged**